### PR TITLE
Fix StyleCop issues that belong to CSharp.Readability

### PR DESCRIPTION
**Pull request by lewischeng-ms**

### CODE REVIEW

File: src/Shared/TypeExtensions.cs [hunk 1]
```diff
@@ -32,5 +40,10 @@ public static Type FindGenericType(this Type type, Type definition)
 
             return null;
         }
+
+        internal static MethodInfo GetQualifiedMethod(this Type type, string methodName)
```
Comment (congysu) [inline src/Shared/TypeExtensions.cs]: `public`? given that the type `TypeExtensions` is `internal`.

File: src/Shared/TypeExtensions.cs [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 // Licensed under the MIT License.  See License.txt in the project root for license information.
 
+using System.Reflection;
+
 namespace System
 {
     internal static partial class TypeExtensions
     {
+        private const BindingFlags QualifiedMethodBindingFlags = BindingFlags.NonPublic |
+                                                                 BindingFlags.Static |
```
Comment (congysu) [inline src/Shared/TypeExtensions.cs]: Is this indentation suggested by StyleCop.  Seems different from method parameters indentation.

Comment (lewischeng-ms) [inline src/Shared/TypeExtensions.cs]: Actually, StyleCop has no preference on the indentation of initialization expressions. Just use this style to make it look prettier than the indentation as parameters.

File: src/Shared/TypeExtensions.cs (hunk 1 above)
Comment (lewischeng-ms) [inline src/Shared/TypeExtensions.cs]: Nice catch. I will update this in some consecutive commit.

**Pull request merged**